content/constants.js

```diff
@@ -6,13 +6,20 @@ require("dotenv").config({
   path: path.join(__dirname, "..", process.env.ENV_FILE || ".env"),
 });
 
-const CONTENT_ROOT = process.env.CONTENT_ROOT;
+let CONTENT_ROOT = process.env.CONTENT_ROOT;
 if (!CONTENT_ROOT) {
   throw new Error("Env var CONTENT_ROOT must be set");
 }
 if (!fs.existsSync(CONTENT_ROOT)) {
   throw new Error(`${path.resolve(CONTENT_ROOT)} does not exist`);
 }
+const nestedFilesDir = path.join(CONTENT_ROOT, "files");
+if (fs.existsSync(nestedFilesDir)) {
```

Comment:
I think it would be more explicit and easier to understand what it does if you add something like:
```
if (path.basename(CONTENT_ROOT) !== 'files' && fs.existsSync(path.join(CONTENT_ROOT, 'files')) {
```
Also, either before or after that line, can you jot down a code comment that explains why we're bothering with this. It doesn't need to explain what the effect is but the motivation that started this "hack".